### DISCUSSION TITLE
Disabling the use of TMDB?

### DISCUSSION THREAD

**Discussion by drHuangMHT** [bitmagnet-io/bitmagnet#233] [Q&A] [answered]
It seems that bigmagnet can't start properly without connection to TMDB. This is really annoying when I happen to be in a country where access to outer Internet is limited. Is there a way to disable the use of TMDB? I can totally live without it.

**Comment by mgdigital** [comment, marked as answer]
Hi, there isn't a way to disable this at the moment, but I'm already aware of this limitation. It's being worked on as part of making the classifier workflow generally more customisable- there should be some good news on this in the next few weeks!

**Comment by drHuangMHT** [comment]
> Hi, there isn't a way to disable this at the moment, but I'm already aware of this limitation. It's being worked on as part of making the classifier workflow generally more customisable- there should be some good news on this in the next few weeks!

Great! Looking forward to it!
